controller/controller.go

```diff
@@ -216,3 +216,11 @@ func (c *Impl) processNextWorkItem() bool {
 
 	return true
 }
+
+// GlobalResync enqueues all objects from the passed SharedInformer
+func (c *Impl) GlobalResync(si cache.SharedInformer) {
+	store := si.GetStore()
+	for _, obj := range store.List() {
+		c.Enqueue(obj)
```

Comment:
Another alternate is to use the si.GetStore().Resync(). Wouldn't that be better?

Reply:
I really like the idea.

The intent is to enqueue all objects that the informer is responsible informing of. Will `si.GetStore().Resync()` do the job?

Reply:
Turns out, that's not the case! A quick manual test shows updates are not happening.
Apparently, we need to make sure to enqueue the listed objects.